lib/commands/streamscreen.js

```diff
@@ -119,7 +120,11 @@ async function initDeviceStreamingProc (adb, deviceInfo, opts = {}) {
   });
 
   let isStarted = false;
-  const deviceStreamingLogger = logger.getLogger(`${SCREENRECORD_BINARY}@${this.opts.udid}`);
+  const deviceStreamingLogger = logger.getLogger(`${SCREENRECORD_BINARY}@` +
+    _.truncate(deviceInfo.udid, {
```

Comment:
This seems like it can be pulled out, since it is used twice in the same way.

Reply:
changed